.circleci/config.yml

```diff
@@ -0,0 +1,57 @@
+jobs:
+  test:
+    executor:
+      name: go/default
+      tag: "1.14"
+    environment:
+      VDLPATH:
+    steps:
+      - checkout
+      - go/load-cache
+      - go/mod-download
+      - run:
+          name: goimports
+          command: |
+            go get golang.org/x/tools/cmd/goimports
+            go install -x golang.org/x/tools/cmd/goimports
+      - go/save-cache
+      - run:
+          name: vdlpath
+          command: |
+            echo "export VDLPATH=$CIRCLE_WORKING_DIRECTORY" >> $BASH_ENV
+            source $BASH_ENV
```

Comment:
Is the `source $BASH_ENV` doing anything? As far as I can tell each step runs in a separate shell.